libs/net/dal/Models/ContentFileReference.cs

```diff
@@ -177,6 +178,7 @@ public ContentFileReference(FileReference fileReference, IFormFile file)
         this.UpdatedById = fileReference.UpdatedById;
         this.UpdatedOn = fileReference.UpdatedOn;
         this.Version = fileReference.Version;
+        this.SourceFile = "";
```

Comment:
This issue was introduced in the Video/Audio editor PR. I'm not certain the best way to resolve it. For now I resolved the linting issue.